AsyncDisplayKit/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -0,0 +1,153 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "_ASDisplayViewAccessiblity.h"
+#import "_ASDisplayView.h"
+#import "ASDisplayNode+FrameworkPrivate.h"
+
+#import <objc/runtime.h>
+#import <queue>
+
+
+#pragma mark - UIAccessibilityElement
+
+static const char *ASDisplayNodeAssociatedNodeKey = "ASAssociatedNode";
+
+@implementation UIAccessibilityElement (_ASDisplayView)
```

Comment:
If you look into UIAccessiblity.h under UIAccessibilityContainer:

>  Sub-elements of a container that are not represented by concrete UIView
 instances (perhaps painted text or icons) can be represented using instances
 of UIAccessibilityElement class (see UIAccessibilityElement.h).

Furthermore take a look into UIAccessibilityElement.h:
>  Instances of this class can be used as "fake" accessibility elements.
 An accessibility container (see UIAccessibility.h) can create and vend instances
 of UIAccessibilityElement to cover for user interface items that are not
 backed by a UIView (for example: painted text or icon).